index.html

```diff
@@ -35,6 +35,90 @@ <h2 class="green">I’m a software developer! </h2></div>
         <li><a href=""><img src="icons/angellist.svg"></a></li>
       </ul>
     </div>
-      </section>
+    <img src="icons/Disabled.svg" alt="next">
+    <div class="works">
+      <h2>My Recent works</h2>
+      <h2><hr></h2>
+      <div class="container1">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="about">
+        <h2>About me</h2><img  class="right-corner" src="icons/Mask Group.svg" alt="">
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.</p>
+      <button type="submit">Get my resume</button><img class="left-corner" src="icons/illustration 2-about me mobile.svg" alt="">
+      </div>
+     <div class="containers">
+      <div class="language">
+        <ul>
+          <li><img src="icons/Languages.svg" alt=""></li>
+          <li><h3>Languages</h3></li>
+          <li>
+            <ol>
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ol>
+          </li>
+        </ul>
+     </div>
+     <div class="container2">
+     <div class="frameworks">
+      <ul>
+        <li><img src="icons/icon-frameworks.svg" alt=""></li>
+        <li><h3>Frameworks</h3></li>
+        <li>
+          <ol>
+            <li>React.js</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ol>
+        </li>
+      </ul>
+   </div>
+  </div>
+  <div class="container3">
+    <div class="skills">
+     <ul>
+       <li><img src="icons/icon-skills.svg" alt="skills"></li>
+       <li><h3>Skills</h3></li>
+       <li>
+         <ol>
+           <li>Database Management</li>
+           <li>Version Control</li>
+           <li>CLI</li>
+           <li>Web Development</li>
+           <li>API Design</li>
+         </ol>
+       </li>
+     </ul>
+  </div>
+  <div class="container4"><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+  <div class="container5">
+    <form action="/action_page.php">
+      <input type="text" id="fname" name="fullname" placeholder="Full name">
+      <input type="text" id="email" name="lastname" placeholder="Your last name..">
+      <textarea placeholder="Hey I would like to connect to ..."></textarea>
+      <button type="submit">Get in touch</button>
+    </form>
+  </div>
+ </div>
+    </div>
+  </div>
```

Comment:
- [ ]  Kindly consider wrapping this whole section in a section tag instead of a div because its a section on the page and doing that will help people navigate around the page with better experience